src/julia_gcext.h

```diff
@@ -25,6 +25,9 @@ typedef void (*jl_gc_cb_post_gc_t)(int full) JL_NOTSAFEPOINT;
 typedef void (*jl_gc_cb_notify_external_alloc_t)(void *addr, size_t size) JL_NOTSAFEPOINT;
 typedef void (*jl_gc_cb_notify_external_free_t)(void *addr) JL_NOTSAFEPOINT;
 
+// Notify if system is under pressure
+typedef void (*jl_gc_cb_notify_gc_pressure_t)(void) JL_NOTSAFEPOINT;
```

Comment:
I think this should go in a separate PR if it is being added and not just moved from somewhere else.

Reply:
Done. https://github.com/JuliaLang/julia/pull/52962.